nbinteract/figure.py

```diff
@@ -0,0 +1,259 @@
+"""
+Methods of the Figure and Plot class to create more complex plots.
+"""
+
+import numpy as np
+import bqplot as bq
+import collections
+import ipywidgets as widgets
+import itertools
+import functools
+import logging
+import toolz.curried as tz
+from IPython.display import display
+from . import util
+from . import plotting
+
+__all__ = ['Hist', 'Bar', 'Scatter', 'Line', 'Plot']
+
+
+def Hist(hist_function, *, options={}):
+    """
+    Public API function to allow users to create a Hist object used to
```

Comment:
nit: No need to say `Public API function to allow users`. That's pretty obvious from the file and function name already.

Reply:
done!